internal/grpctest/grpctest.go

```diff
@@ -53,13 +55,18 @@ func RunSubTests(t *testing.T, x interface{}) {
 	setup := getTestFunc(t, xv, "Setup")
 	teardown := getTestFunc(t, xv, "Teardown")
 
+	logger.setLevel(infoLog)
+	logger.setVerbosity(2)
```

Comment:
Can you see if this value (2) affects the runtime of the tests vs. 0?

I'm thinking our test logger should honor the environment variable `GRPC_GO_LOG_VERBOSITY_LEVEL`, to keep things easier to remember.

Reply:
I don't notice a difference in execution time of the tests. Not sure if granularity deeper than fractions of a second is needed. The verbosity level now respects the `GRPC_GO_LOG_VERBOSITY_LEVEL` environment variable.

Reply:
> I don't notice a difference in execution time of the tests. Not sure if granularity deeper than fractions of a second is needed. The verbosity level now respects the GRPC_GO_LOG_VERBOSITY_LEVEL environment variable.

That should be good enough.  Let's try using a default of 2, then, so we'll have more information available when things fail.  If it turns out to be too much, we can tone it down.